apps/web/src/components/PaginatedTable/index.tsx

```diff
@@ -120,6 +126,9 @@ export const PaginatedTable: FC<PaginatedTableProps> = function ({
         emptyState={DEFAULT_TABLE_EMPTY_STATE}
       >
         <div className="flex flex-col gap-6">
+          <div className="w-1/2">
+            <Filters />
+          </div>
           <Table
```

Comment:
Can we move the `Filters` out of  the `PaginatedTable` component?

Let's have them wrapped within their own `Card` component on top of the table card

Reply:
It should also take the whole width